governance/roles.md

```diff
@@ -159,9 +172,12 @@ goals, the meeting facilitator has the following responsibilities:
 Prerequisites:
 * active member
 * history of regular attendance
-* prior participation as scribe
+
+Preferred prerequisites:
```

Comment:
The position of this is confusing.  I think "One or more of the following" is requirement, not "preferred"
Original thinking was to get people involved and provide evidence that they will "chop wood / carry water", but I would be ok with removing scribe requirement, since the below items are significant